action/ls.go

```diff
@@ -19,7 +19,7 @@ var ownerID string
 
 // ListHandler is all params for List func
 type ListHandler struct {
-	*FlagHandler
+	BaseHandler
 	// Remote is the remote qs path
 	Remote string `json:"remote"`
```

Comment:
Maybe we don't need to export them?

Reply:
> Maybe we don't need to export them?

If we want to support introducing these params by different means, we have to export them.